docs/admin/access.rst

```diff
@@ -159,16 +169,6 @@ Custom access control
 
 .. include:: /snippets/not-hosted-libre.rst
 
-.. versionchanged:: 3.0
-
-    Before Weblate 3.0, the privilege system was based on Django, but is now
-    specifically built for Weblate. If using anything older, please consult
-    the documentation for the specific version you are using.
-
-The permission system is based on groups and roles, where roles define a set of
-permissions, and groups link them to users and translations, see
-:ref:`auth-model` for more details.
-
```

Comment:
I've moved this here, because this is irrelevant for the `Simple access control`, because user have no interaction with roles or permissions there, so mentioning it in the chapter's opening looks very confusing.

Reply:
@orangesunny, I didn't reverted your commit, but I believe it should be, your thoughts?

Reply:
You are right, I put it partially back, please check: https://github.com/WeblateOrg/weblate/pull/5566/commits/5ce371a0df79cbaf121f1665a7183c44259f1fdf